From 19c3bec6e6b8793db488f6efeac80365bd39f7a8 Mon Sep 17 00:00:00 2001
From: Stefan Christ <contact@stefanchrist.eu>
Date: Mon, 14 Nov 2016 00:03:12 +0100
Subject: [PATCH 1892/4301] drm/fb-helper: add fb_debug_* to
 DRM_FB_HELPER_DEFAULT_OPS

commit 87a072f10e3fd709330d9382e48e7292c62e63be from
git://git@github.com:01org/linux-intel-4.9.git

Add additional members fb_debug_enter and fb_debug_leave to helper
define. They are shared by all fb_ops implementations.

Suggested-by: Daniel Vetter <daniel@ffwll.ch>
Signed-off-by: Stefan Christ <contact@stefanchrist.eu>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Link: http://patchwork.freedesktop.org/patch/msgid/1479078208-25221-3-git-send-email-contact@stefanchrist.eu
---
 include/drm/drm_fb_helper.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/include/drm/drm_fb_helper.h b/include/drm/drm_fb_helper.h
index ed8edfe..975deed 100644
--- a/include/drm/drm_fb_helper.h
+++ b/include/drm/drm_fb_helper.h
@@ -228,7 +228,9 @@ struct drm_fb_helper {
 	.fb_set_par	= drm_fb_helper_set_par, \
 	.fb_setcmap	= drm_fb_helper_setcmap, \
 	.fb_blank	= drm_fb_helper_blank, \
-	.fb_pan_display	= drm_fb_helper_pan_display
+	.fb_pan_display	= drm_fb_helper_pan_display, \
+	.fb_debug_enter = drm_fb_helper_debug_enter, \
+	.fb_debug_leave = drm_fb_helper_debug_leave
 
 #ifdef CONFIG_DRM_FBDEV_EMULATION
 void drm_fb_helper_prepare(struct drm_device *dev, struct drm_fb_helper *helper,
-- 
1.7.5.4

